cpp/include/cudf/copying.hpp

```diff
@@ -717,5 +717,47 @@ std::unique_ptr<scalar> get_element(
   size_type index,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Indicates whether a row can be sampled more than once.
+ **/
+enum class row_multi_sampling : bool {
+  DISALLOWED,  // A row can be sampled only once
```

Comment:
If you take my suggestion above, I would use "TRUE" and "FALSE". or "ON" and "OFF".